data/timeline.json.unvalidated.in

```diff
@@ -958,34 +957,788 @@
+                            "yes": {
+                                "text": "Ready!"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "yes": [
+                        "chat::ada::processing::wave_stroke_blending",
+                        "chat::ada::processing::wave_stroke_purple",
+                        "chat::ada::processing::wave_rgb_moreinfo",
+                        "chat::ada::processing::wave_rgb_moreinfo_pause"
+                    ]
+                }
+            }
+        },
+        {
+          "name": "chat::ada::processing::wave_stroke_blending",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Now let’s try blending our RGB values: stroke(0,255,255)"
+          }
+        },
+        {
+          "name": "chat::ada::processing::wave_stroke_purple",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Looks like green and blue make a lovely teal color! Why not explore and try reducing the numbers to see what happens. I love this purple color: (150,0,255)"
+          }
+        },
+        {
+          "name": "chat::ada::processing::wave_rgb_moreinfo",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Play around with the color parameters and press Ready to learn how to change the colors by pressing the mouse. To learn more about working with RGB values, visit this website: <a href=\"http://web.stanford.edu/class/cs101/image-rgb-explorer.html\">Image RGB Explorer</a>"
+          }
+        },
+        {
+            "name": "chat::ada::processing::wave_rgb_moreinfo_pause",
+            "type": "input-user",
+            "data": {
+                "actor": "Ada",
+                "input": {
+                    "type": "choice",
+                    "settings": {
+                        "prompt": "Ready to continue?",
+                        "choices": {
+                            "yes": {
+                                "text": "Ready!"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "yes": [
+                        "chat::ada::processing::wave_if",
+                        "chat::ada::processing::wave_stroke_if",
+                        "chat::ada::processing::wave_stroke_sample_code",
+                        "chat::ada::processing::wave_stroke_sample_code_pause"
+                    ]
+                }
+            }
+        },
+       {
+          "name": "chat::ada::processing::wave_if",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Welcome back! Time to get interactive. Let’s work with the “If” command to choose our bright red color only when the mouse is pressed."
```

Comment:
Should 'If' be capitalised? Also, find-and-replace UTF-8 quotes and replace with ASCII quotes.